<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Page Setting</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_x">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_y">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="le_width">
           <property name="text">
            <string>768</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="le_height">
           <property name="text">
            <string>1024</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Tool</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="6" column="0">
          <widget class="QToolButton" name="toolButton_7">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QToolButton" name="btnCheckbox">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="app.qrc">
             <normaloff>:/app/img/par_1411128613.jpg</normaloff>:/app/img/par_1411128613.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="btnText">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="app.qrc">
             <normaloff>:/app/img/red-label.jpg</normaloff>:/app/img/red-label.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="btnRadiobox">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="app.qrc">
             <normaloff>:/app/img/custom-radio-buttons.png</normaloff>:/app/img/custom-radio-buttons.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="btnCombo">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="app.qrc">
             <normaloff>:/app/img/combobox-data-entry-s.png</normaloff>:/app/img/combobox-data-entry-s.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="btnBarcode">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="app.qrc">
             <normaloff>:/app/img/barcode.png</normaloff>:/app/img/barcode.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="toolButton_6">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tbProperty">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Item</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Title</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>Canada</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string>AM</string>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string>China</string>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string>5</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnApply">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
       <item row="0" column="1">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>298</width>
            <height>566</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
